srcjs/app.js

```diff
@@ -1,4 +1,10 @@
+import Framework7 from 'framework7/bundle'
+import { getAppInstance } from './init.js'
```

Comment:
Not 100% sure if this is the correct approach yet, might be annoying once we would modularize app.js even further

Reply:
Storing in window?

Reply:
Personally I'm not a fan of using window because it's harder to track down where it comes from and global namespace pollution and such, but we could do in this case, it's just one object that applies to basically everything you do within Framework7. So, I think 3 options:

1. Set `window.app` like you suggest
2. Define imports like so, at the risk of annoying devs, but personally I feel like this is the most correct way (?):
```
import { getAppInstance } from './init.js';

const app = getAppInstance();
```
3. Create a helper function, still at the risk of annoying devs, but with less typing 😜 :

```
import { getAppInstance } from './init.js';

export function withApp(callback) {
  const app = getAppInstance();
  callback(app);
}
```
and import like so:
```
import { withApp } from './appUtils.js';

withApp((app) => {
  // Do stuff here with app
});
```

I'm going with 2 for now, but changing it will be straightforward. Big commit incoming 😁